src/commands/execute.rs

```diff
@@ -0,0 +1,346 @@
+//! Module for wax, executes a module immediately
+
+use crate::data::wax_index;
+use crate::dataflow::find_command_result::FindCommandResult;
+use crate::dataflow::installed_packages::{InstalledPackages, RegistryInstaller};
+use crate::dataflow::lockfile_packages::LockfileResult;
+use crate::dataflow::resolved_packages::ResolvedPackages;
+use crate::dataflow::WapmPackageKey;
+use crate::graphql::{execute_query, DateTime};
+use crate::keys;
+use crate::util;
+
+use graphql_client::*;
+
+use std::convert::From;
+use std::env;
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::str::FromStr;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+pub struct ExecuteOpt {
+    /// Command name
+    command: String,
+    /// Run unsandboxed emscripten modules too
+    #[structopt(long = "emscripten")]
+    run_emscripten_too: bool,
```

Comment:
Somehow I don't like the naming of this variable. The `*_too` implies a side effect.

I think a name like: `enable_emscripten` might be better